### ISSUE TITLE
bug: checksum mismatch for k8s.io/client-go

### ISSUE THREAD

**Issue by radu-matei** [slok/brigadeterm#53]
Hi!
I'm trying to build from source and currently getting a dependency resolution error for `k8s.io/client-go`:

```
$ go version
go version go1.11.1 linux/amd64

$ GO111MODULE=on go build ./cmd/brigadeterm/
go: downloading k8s.io/client-go v2.0.0-alpha.0.0.20181016174657-85ed251159e4+incompatible
go: downloading github.com/spf13/pflag v1.0.0
go: downloading gopkg.in/inf.v0 v0.9.0
go: finding github.com/gogo/protobuf/sortkeys latest
go: downloading golang.org/x/text v0.3.0
go: verifying k8s.io/client-go@v2.0.0-alpha.0.0.20181016174657-85ed251159e4+incompatible: checksum mismatch
        downloaded: h1:lSH23zhtRXjVkgLDT0lWAgIXh2G2LtG+FyusOC1fXl4=
        go.sum:     h1:8KU/0fmVnT0P/iIBCSYK/fFDI46a5LpShNrBlMI4Wec=
```

I haven't tried to use Go modules with Kubernetes packages (and don't have that much experience with Go modules altogether), so I could be doing something stupid here.

**Comment by slok** [comment]
Hi @radu-matei!

This is due to a [bug in Gomod](https://github.com/golang/go/issues/27925). In order to fix need to use >= `go 1.11.2` and run `go clean -modcache` to generate a new cache.

**Comment by radu-matei** [comment]
Ok, thanks a lot - we can then close this.
